DEVELOPMENT.md

```diff
@@ -0,0 +1,28 @@
+# Preparation
+
+## Install extensions
+   - [Bazel](https://marketplace.visualstudio.com/items?itemName=BazelBuild.vscode-bazel)
```

Comment:
Do we actually use it for any task now?

Reply:
Yes, we will need to review packages dependencies. This extension make bazel files more readable.